source/RialtoGStreamerMSEAudioSink.cpp

```diff
@@ -553,6 +604,45 @@ static void rialto_mse_audio_sink_set_property(GObject *object, guint propId, co
         }
         break;
     }
+    case PROP_AUDIO_FADE:
+    {
+        const gchar *audioFadeStr = g_value_get_string(value);
+        uint32_t fadeVolume = static_cast<uint32_t>(kDefaultVolume * 100);
+        uint32_t duration = kDefaultVolumeDuration;
+        int easeTypeInt = convertEaseTypeToInt(kDefaultEaseType);
+
+        int parsedItems = sscanf(audioFadeStr, "%u,%u,%d", &fadeVolume, &duration, &easeTypeInt);
+
+        if (parsedItems == 0)
+        {
+            GST_ERROR_OBJECT(object, "Failed to parse any values from audio fade string: %s.", audioFadeStr);
+            return;
+        }
+        else if (parsedItems == 1 || parsedItems == 2)
+        {
+            GST_WARNING_OBJECT(object, "Partially parsed audio fade string: %s. Continuing with values: fadeVolume=%u, duration=%u, easeTypeInt=%d",
+                               audioFadeStr, fadeVolume, duration, easeTypeInt);
+        }
+
+        double volume = fadeVolume / 100.0;
```

Comment:
Maybe we should make volume clipping here?
If fadeVolume > 100, then clip it to 100 and print warning.